src/envo/Makefile

```diff
@@ -210,6 +211,42 @@ subsets/envo-basic.obo: envo-simple.owl
 #pato-lite.obo:
 #	$(OWLTOOLS) $(OBO)/pato.obo --make-subset-by-properties -o -f obo $@
 
+subsets/biome-hierarchy.owl: envo.owl
+	robot filter --input $< \
+	--term ENVO:00000428  \
+	--term IAO:0000115  \
+	--select "annotations self descendants" \
+	--signature true \
+	--output $@
+.PRECIOUS: subsets/biome-hierarchy.owl
+
+subsets/feature-hierarchy.owl: envo.owl
+	robot filter --input $< \
+	--term ENVO:01000813  \
+	--term IAO:0000115  \
+	--select "annotations self descendants" \
+	--signature true \
+	--output $@
+.PRECIOUS: subsets/feature-hierarchy.owl
```

Comment:
No all valid "features" (a concept which is no longer valid in MIxS v5 anyway) will be under astronomical body part (many manufactured objects, for example)

I don't think we should call this the feature hierarchy. Maybe create exports for astronomical body parts, manufactured objects, and other popular main branches?